record.h

```diff
@@ -77,6 +81,11 @@ janus_recorder *janus_recorder_create(const char *dir, const char *codec, const
  * @param[in] length The frame data length
  * @returns 0 in case of success, a negative integer otherwise */
 int janus_recorder_save_frame(janus_recorder *recorder, char *buffer, uint length);
+/*! \brief Pause/Resume the recorder
+ * @param[in] recorder The janus_recorder instance
+ * @param[in] boolean Whether to pause or resume
+ * @returns 0 in case of success, a negative integer otherwise */
+int janus_recorder_pause(janus_recorder *recorder, gboolean pause);
```

Comment:
I'd rather have two separate methods, `janus_recorder_pause` and `janus_recorder_resume`, as from a semantic perspective a method called `janus_recorder_pause` that can be used to resume as well is counter-intuitive.